l1-contracts/script/StakingAssetHandler.s.sol

```diff
@@ -28,6 +28,8 @@ contract StakingAssetHandlerScript is Test {
   string internal constant SCOPE = "testnet.aztec.network";
   string internal constant SUBSCOPE = "personhood";
 
+  bytes32 public constant DEPOSIT_MERKLE_ROOT = bytes32(0);
```

Comment:
![team fortress 2 GIF](https://media3.giphy.com/media/dno0yUmnYsFR6/giphy.gif)

Reply:
Actually I can use the get root script here